tools/api-markdown-documenter/src/LintApiModel.ts

```diff
@@ -21,12 +22,13 @@ import {
 	DocNodeKind,
 } from "@microsoft/tsdoc";
 import { defaultConsoleLogger } from "./Logging.js";
-import { resolveSymbolicReference } from "./utilities/index.js";
+import { resolveSymbolicReference, type SymbolicReferenceError } from "./utilities/index.js";
 import type { ConfigurationBase } from "./ConfigurationBase.js";
 
 /**
  * {@link lintApiModel} configuration.
  *
+ * @sealed
```

Comment:
We don't expect anyone to craft/extend these types, only read them. So they should have all been `@sealed`.